documentation/docs/get-started/api-server-get-started/basic-operations.md

```diff
@@ -2,99 +2,191 @@
 sidebar_label: 'Basic Operations'
 sidebar_position: 3
 ---
+
 # Basic Operations
+
 Let's use the API docs at [`http://localhost:3332/api-docs`](http://localhost:3332/api-docs) to try some basic functions.
+
+## Authentication
+
+Based on your agent configuration, you may need to provide an API key to access the API, the configuration for authentication middleware plugin [here](/docs/agent-configuration/config-agent-file#authentication-middleware-plugin). If you have an API key, you can add it to the request header for every requests like this:
```

Comment:
the last sentence should be "If you have an API key, you can add it to the request header for every request like this" (change requests to request)